ports/esp32/boards/LOLIN_S2_MINI/modules/s2mini.py

```diff
@@ -0,0 +1,34 @@
+# LOLIN S2 MINI MicroPython Helper Library
+
+
+from micropython import const
+from machine import Pin, SPI, ADC
+import machine, time
```

Comment:
unless there's a reason to include them, I'd suggest removing imports that are unused: SPI, ADC, machine, time